sdk/identity/Azure.Identity/src/TokenCacheRefreshArgs.cs

```diff
@@ -26,9 +26,15 @@ public class TokenCacheRefreshArgs
         /// </summary>
         public string SuggestedCacheKey { get; }
 
-        internal TokenCacheRefreshArgs(TokenCacheNotificationArgs args)
+        /// <summary>
+        /// Whether or not the cache is enabled for CAE.
```

Comment:
Should we give some guidance here that CAE enabled cache needs to be stored separately from non-CAE cache?

Reply:
Added some detail